lib/cartopy/mpl/ticker.py

```diff
@@ -22,7 +22,7 @@ class _PlateCarreeFormatter(Formatter):
 
     _target_projection = ccrs.PlateCarree()
 
-    def __init__(self, direction_label=True, degree_symbol='\u00B0',
+    def __init__(self, direction_label=True, degree_symbol='°',
```

Comment:
Hmm, maybe we should use real prime glyphs for minutes and seconds.

Reply:
I think it would make sense to change those to the proper symbols, but that may require some updates to images too so perhaps you want to save that for another PR?

Reply:
It fortunately appears that we don't need to update anything.

Reply:
Fortunately, or unfortunately... This could be because our tolerances are so large like you found in your last PR :)
Feel free to self-merge if you like this as-is.

Reply:
Heh, yes, but I don't think any tests have ticks smaller than degrees. Fortunately, they are checked text-wise in `test_ticker.py`.